Makefile

```diff
@@ -24,7 +24,7 @@
 		fossa
 all: build
 
-build:  wasp
+build:  wasp manifest-generator
 
```

Comment:
I think that this is already done in `make manifests`

Reply:
Yeah I wanted to have a single make target that will build the product code and the tooling, otherwise if I want to catch compilation errors in produce code and in tools code I have to run `make build` and make manifests`

Reply:
Thank you for the explanation; that sounds good to me.